handlers/postitem/postItem.go

```diff
@@ -0,0 +1,71 @@
+package postitem
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/mount-joy/thelist-lambda/data"
+	"github.com/mount-joy/thelist-lambda/db"
+	"github.com/mount-joy/thelist-lambda/handlers/iface"
+)
+
+type postItem struct {
+	db db.DB
+}
+
+// New returns an instance of postItem satisfying the RouteHandler interface
+func New() iface.RouteHandler {
+	return &postItem{
+		db: db.DynamoDB(),
+	}
+}
+
+// Match returns true if this RouteHandler should handle this request
```

Comment:
Wonder if in all of them this should be a little more specific - no reason it wouldn't mean that this is for post item requests
Although the name of `postItem` maybe is explicit enough